tests/ui_suite/fav_test.py

```diff
@@ -1,44 +1,39 @@
 import pytest
+import allure
 import pytest_check as check
-from tests.pages.pageobjects import LoginedPage
+from tests.pages.pageobjects import LoginedPage, MainPage
 import tests.common as common
 
-"""параметризуем массивом индексов товаров и признаком,
-добавляем в избранное из списка или из товара"""
-
 
+@allure.title('Adding some goods to favorites when user is logined')
 @pytest.mark.ui
-@pytest.mark.parametrize('fav_numbers, add_fav_from_detail', (
-        ((0, 1), 'add_from_goods_list'),
-        ((0, 2, 4), 'add_from_goods_list'),
-        ((2, 5), 'add_from_goods_detail')))
+@pytest.mark.parametrize('fav_numbers, add_fav_from_detail',
+                         (
+                                 ((0, 1), 'add_from_goods_list'),
```

Comment:
as for me the previous formating looks better )

Reply:
returned previous variant